### ISSUE TITLE
getting error when trying to run

### ISSUE THREAD

**Issue by mjsmjs** [SenorSmartyPants/Comixology-Scraper#10]
I get this on the first attempt

cYo\ComicRack\Scripts\Comixology Scraper\ComixologyCR.py'
Could not add reference to assembly HtmlAgilityPack

Then this on the second

'ScopeStorage' object has no attribute 'ComixologyScraper'

**Comment by SenorSmartyPants** [comment]
First. This plugin will not run properly at the moment because I have not updated it to use the new Amazon Comixology website.

Did you download from the Comicrack branch? https://github.com/SenorSmartyPants/Comixology-Scraper/archive/refs/heads/ComicRack.zip

**Comment by mjsmjs** [comment]
Yes I used the comicrack branch. But will now use the amazon one. Thank you!
